src/libraries/Common/src/System/Runtime/InteropServices/LibraryImportAttribute.cs

```diff
@@ -17,14 +17,14 @@ namespace System.Runtime.InteropServices
 #else
     internal
 #endif
-    sealed class GeneratedDllImportAttribute : Attribute
+    sealed class LibraryImportAttribute : Attribute
```

Comment:
Sad that LIBRARYIMPORT_GENERATOR_TEST cant be removed and make this public for .NET 7+ (and the generator that works for it) as part of the .NET SDK.

Reply:
That will be coming (but not in this PR).

Reply:
@AraHaan See https://github.com/dotnet/runtime/issues/60595